### PR TITLE
bazarr: update to 1.4.0

### PR DESCRIPTION
**Pull request by smaarn**
## Description

* Upgrades bazarr version to 1.4.0
* Cleanup no longer relevant `ARMv7L_ARCHS` logics for numpy

## Checklist

- [x] Build rule `all-supported` completed successfully
- [x] New installation of package completed successfully
- [x] Package upgrade completed successfully (Manually install the package again)
- [x] Package [functionality was tested](https://github.com/SynoCommunity/spksrc/wiki/Package-Update-Policy#tests-checks)

### Type of change

- [x] Package update

### CODE REVIEW

**Comment by ngthwi** [comment]
Working fine on a DS215j and a DS218j
Thanks a lot

**Comment by hgy59** [comment]
> Working fine on a DS215j and a DS218j Thanks a lot

Those models are not ARMv7L

Only models with hi3535 CPU are ARMv7L, i.e.
- Routers (SRM 1.x)
- Network Video Recorders (NVR216, NVR1218)

Details see https://github.com/SynoCommunity/spksrc/wiki/Synology-and-SynoCommunity-Package-Architectures

**Comment by smaarn** [comment]
Flagging as draft as, even if it does compile and build successfully we don't really have any client for testing it, as of now.

**Comment by ngthwi** [comment]
This is the status page on a DS218j...
IMG_6450

**Comment by smaarn** [comment]
@ngthwi I think this ought to be ready for merge & release. Do you think you could have time to test the latest 1.4.0 binaries ? They are available here: https://github.com/SynoCommunity/spksrc/pull/5911/checks 

@th0ma7 any objection to merging ? (things I wouldn't have spotted for example...)

All apologies for the delays

**Comment by hgy59** [comment]
Just to be clear: `$(ARMv7L_ARCHS)` is not related to `armv7l` as reported by `uname -m`.

we have the definitions in `spksrc.archs.mk`
```
ARMv7_ARCHS = $(GENERIC_ARMv7_ARCH) $(DSM_ARMv7_ARCHS) $(SRM_ARMv7_ARCHS)
# hi3535 is not supported by generic ARMv7 arch
ARMv7L_ARCHS = hi3535
```

`ARMv7L_ARCHS` is used to separate `hi3535` cpu from other armv7 cpus.

None of the Diskstation Models have `ARMv7L_ARCHS`. Only the (Network Video Recorder) Models NVR1218 and NVR216 and the (VideoStation) Model VS360HD have.

So the title of this PR is missleading since armv7 support for DSM was never dropped.

Was there any request to support this package on  NVR1218, NVR216 or VS360HD?.
If not, I recommend to keep
```
UNSUPPORTED_ARCHS = $(ARMv5_ARCHS) $(ARMv7L_ARCHS)
```
and to remove the `<br />2. Add back support for ARMv7L architectures'` from the changelog.

If you want to keep the ARMv7L_ARCHS then please update the changelog to 
 `<br />2. Add back support for NVR1218, NVR216 and VS360HD Models'`

**Comment by smaarn** [comment]
> Just to be clear: `$(ARMv7L_ARCHS)` is not related to `armv7l` as reported by `uname -m`.
> 
> we have the definitions in `spksrc.archs.mk`
> 
> ```
> ARMv7_ARCHS = $(GENERIC_ARMv7_ARCH) $(DSM_ARMv7_ARCHS) $(SRM_ARMv7_ARCHS)
> # hi3535 is not supported by generic ARMv7 arch
> ARMv7L_ARCHS = hi3535
> ```
> 
> `ARMv7L_ARCHS` is used to separate `hi3535` cpu from other armv7 cpus.
> 
> None of the Diskstation Models have `ARMv7L_ARCHS`. Only the (Network Video Recorder) Models NVR1218 and NVR216 and the (VideoStation) Model VS360HD have.
> 
> So the title of this PR is missleading since armv7 support for DSM was never dropped.
> 
> Was there any request to support this package on NVR1218, NVR216 or VS360HD?. If not, I recommend to keep
> 
> ```
> UNSUPPORTED_ARCHS = $(ARMv5_ARCHS) $(ARMv7L_ARCHS)
> ```
> 
> and to remove the `<br />2. Add back support for ARMv7L architectures'` from the changelog.
> 
> If you want to keep the ARMv7L_ARCHS then please update the changelog to `<br />2. Add back support for NVR1218, NVR216 and VS360HD Models'`

I think I got confused here with the custom logic for numpy which was related to those `ARMv7L_ARCHS` architectures.

Will be reverting the change **and** removing the therefore useless tricks for numpy.

File: spk/bazarr/Makefile [hunk 1]
```diff
@@ -65,11 +65,8 @@ else
 WHEELS += src/requirements-crossenv-greenlet-v1.txt
 endif
 
-# [numpy] <= 1.22.4 (armv7l)
-ifeq ($(findstring $(ARCH),$(ARMv7L_ARCHS)),$(ARCH))
-WHEELS += src/requirements-crossenv-numpy-armv7l.txt
```
Comment (smaarn) [inline spk/bazarr/Makefile:70]: It seems that `ARMv7L_ARCHS` are actually part of the unsupported architectures and this line is therefore useless

**Review by hgy59 — APPROVED** [review]
LGTM

**Comment by ngthwi** [comment]
> @ngthwi I think this ought to be ready for merge & release. Do you think you could have time to test the latest 1.4.0 binaries ? They are available here: https://github.com/SynoCommunity/spksrc/pull/5911/checks
> 
> @th0ma7 any objection to merging ? (things I wouldn't have spotted for example...)
> 
> All apologies for the delays

Already tested that...
Working fine

**Comment by mreid-tt** [comment]
Hello @smaarn, I've successfully published the builds for the merged PR on the website. The `qoriq` build remains inactive, due to a compatibility issue with OpenSSL 3, as reported in https://github.com/SynoCommunity/spksrc/pull/5905#issuecomment-1740656871.

By the way, I've observed that the recent commits you've merged were published by other contributors. While we're more than willing to provide assistance, if you encounter any challenges with your publishing process, feel free to reach out to me on Discord for support.

**Pull request merged**